testdata/baselines/reference/submodule/compiler/importHelpersVerbatimModuleSyntax.js.diff

```diff
@@ -5,9 +5,7 @@
 
  //// [main.cjs]
 -const tslib_1 = require("tslib");
++import tslib_1 = require("tslib");
```

Comment:
IMO, this is obviously not right - even in `verbatimModuleSyntax`, you shouldn't have TS syntax in the output - but it's not the fault of this PR. `module: preserve` seems to be broken with `importHelpers` right now, and adding the helper to this test just triggers the broken behavior.